### PR TITLE
New Config

### PR DESCRIPTION
**Pull request by mayabrandi**
Assuming MONGO_URI and MONGO_DBNAME as os.environ variables!

Thanks! @ingkebil

### CODE REVIEW

File: instance/config.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-SECRET_KEY = 'MySuperSecretKey'
+# mongo
+DB_URI = os.environ['MONGO_URI']
+DB_NAME = os.environ['MONGO_DBNAME']
 
-DB_URI =  "mongodb://localhost:27017"
-DB_NAME = 'trending'
\ No newline at end of file
+DEBUG = True
+SECRET_KEY = 'MySuperSecretKey'
```
Comment (ingkebil) [inline instance/config.py]:
```suggestion
SECRET_KEY = os.environ['VOGUE_SECRET_KEY']
```

File: instance/config.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 
-SECRET_KEY = 'MySuperSecretKey'
+# mongo
+DB_URI = os.environ['MONGO_URI']
+DB_NAME = os.environ['MONGO_DBNAME']
 
-DB_URI =  "mongodb://localhost:27017"
-DB_NAME = 'trending'
\ No newline at end of file
+DEBUG = True
```
Comment (ingkebil) [inline instance/config.py]:
```suggestion
DEBUG = os.environ['VOGUE_DEBUG']
```

Comment (ingkebil) [reply instance/config.py]: ... so we can set them in the shell env instead of hardcoding them in the config

**Comment by mayabrandi** [comment]
I commited your suggested changes @ingkebil !

Is it okej now?

**Pull request merged**